src/js/messaging/containers/Thread.jsx

```diff
@@ -277,18 +268,18 @@ export class Thread extends React.Component {
   }
 
   render() {
-    if (this.props.isNewMessage && this.props.loadingRecipients) {
-      return <LoadingIndicator message="Loading the application..."/>;
-    }
-
     const loading = this.props.loading;
```

Comment:
Hm, that would be cleaner. I often forget that's a thing. I can certainly change it to that.